templates/_partials/form-fields.tpl

```diff
@@ -129,6 +128,7 @@
           <input
             class="form-control js-child-focus js-visible-password"
             name="{$field.name}"
+            id="{$field.name}"
```

Comment:
IMO it's too generic. We're talking about something like `id="phone"`. Someone may use that in other context (eg. in contact information) and add special style to `#phone` which will also impact this form field.

`#field-firstname` `#field-phone` `#field-gdpr` etc. should be ok.

Reply:
I agree, and it is like that in `classic-theme`, https://github.com/PrestaShop/classic-theme/blob/develop/templates/_partials/form-fields.tpl#L34

Reply:
You're right!